tests/unit/conftest.py

```diff
@@ -131,6 +179,65 @@ def single_passing_xml(tmpdir_factory,
     return filename
 
 
+@pytest.fixture(scope='session')
+def invalid_config_file(tmpdir_factory,
+                        default_global_properties,
+                        default_testcase_properties,
+                        default_testcase_elements):
+    """JUnitXML sample representing a single passing test."""
+
+    filename = tmpdir_factory.mktemp('data').join('config_file.xml').strpath
+    config_json = \
+        """{
+             "environment_variables": {
+               "BUILD_URL": "url",
+           """
+
+    with open(filename, 'w') as f:
+        f.write(config_json)
+
+    return filename
+
+
+@pytest.fixture(scope='session')
+def valid_config_file(tmpdir_factory,
```

Comment:
This is a 'pytest-zigzag-config' file. The name and description of the fixture is misleading.